ProjectFiles/Assets/Scripts/Game/RoundManager.cs

```diff
@@ -101,14 +98,21 @@ public void StartPreRound()
 
         private void StartRoundEvent()
         {
-            // TODO: give an actual dynamic value to num space
+            int numberOfSpaces = 1;
+            // If not the last round
+            if (roundNumber < RoundProperties.MaxRounds - 1)
+            {
+                numberOfSpaces = (int) Math.Ceiling(world.GetNumPlayers() * RoundProperties.SpacesToPlayersRatio);
```

Comment:
Doesn't work with 2 players. 

ceil(2 * 0.6) = 2 so there's always two spaces for two player
```suggestion
                numberOfSpaces = (int) Math.Floor(world.GetNumPlayers() * RoundProperties.SpacesToPlayersRatio);
```